packages/shared/src/components/modals/ShareModal.tsx

```diff
@@ -96,7 +101,7 @@ export default function ShareModal({
         />
         <p className="py-2.5 font-bold typo-callout">Share via</p>
         <SocialShare
-          post={post}
+          post={article}
```

Comment:
Why do we moved to article here?
Would sharedPost not be more descriptive?

Reply:
Sab has decided when you are sharing an article from the feed, it must be pointed to the original article. Relevant discussion: https://dailydotdev.slack.com/archives/C03QZBF46JC/p1675073160566779?thread_ts=1675068777.979039&cid=C03QZBF46JC

The variable article can either be the original post as the fallback value. `sharedPost` is reserved for posts inside a Squad and this can be either of the two.

Reply:
I am with @sshanzel on this one. Article makes more sense since it would mean the post for the regular posts, and sharedpost for the squad posts. Which is the article for both cases.